### ISSUE TITLE
[Suggestion]: Better Mac Support

### ISSUE THREAD

**Issue by souldbminerr** [nonamecrackers2/simple-clouds#12]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

Looking at the OpenGL requirements, this won't work on M1 Macs, due to them only supporting up to OGL 4.1 (after that metal must be used)

**Comment by nonamecrackers2** [comment]
Yeah I haven't had much of an ability to test on macs since I personally don't have one. I'll have to see if I can find one to dev on it.

I'll have to figure out extensions/stuff to get compute shaders working if M1 macs are stuck on 4.1

**Comment by gedsweyerv** [comment]
did you ever find a way?

**Comment by nonamecrackers2** [comment]
It doesn't seem possible, even with extensions unfortunately.
